### ISSUE TITLE
NTR: Phosphoenolpyruvate carboxykinase deficiency in fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9099]
New term request Phosphoenolpyruvate carboxykinase deficiency in fibroblasts (segal_071120200401): PEPCK  UMLS: 3279334

**Comment by kanems** [comment]
FYI the recommend CUI above is specific for the 'cytosolic' form of the deficiency, which I was going to recommend UMLS merge with the OMIM record (https://www.omim.org/entry/261680) on C5574905
![image](https://user-images.githubusercontent.com/71083858/209151621-c40bfa06-b571-4237-a65c-c45297601081.png)


If the PEPCK deficiency concept is to be added to HPO, would this broader concept in UMLS be a better match?
Phosphoenolpyruvate carboxykinase deficiency UMLS CUI: C0268194

**Comment by MickeySegal** [comment]
This is for https://omim.org/entry/261680, though the UMLS entry is for the disease, not the lab abnormality, but we included it because it was close enough to be useful.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000619
- label: Reduced phosphoenolpyruvate carboxykinase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:107509
- def: Acitivity of phosphoenolpyruvate carboxykinase (EC 4.1.1.32) is below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9099
- synonyms:
